.env.e2e

```diff
@@ -1,5 +1,5 @@
 DEBUG_WEBVIEW=1
-DEFAULT_NETWORK=goerli_rpc
+DEFAULT_NETWORK=
```

Comment:
@qfrank can you check this pr. It's removing Goerli so I think I might need to make some adjustments on how this DEFAULT NETWORK is used as afaik it affects syncing. 
Let me know if I should make some adjustments

Reply:
Not sure how `CurrentNetwork` is used.

I can only add that it's set here during `CreateAccountAndLogin`:
https://github.com/status-im/status-go/blob/1ba4b86c7e2a63864aa84698039e7a1d46082701/api/geth_backend.go#L1590

But the `CreateAccount.CurrentNetwork` value is deprecated:
https://github.com/status-im/status-
go/blob/1ba4b86c7e2a63864aa84698039e7a1d46082701/protocol/requests/create_account.go#L66-L68

And `CreateAccount.NetworkID` (which is the replacement) is properly used here:
https://github.com/status-im/status-go/blob/1ba4b86c7e2a63864aa84698039e7a1d46082701/api/defaults.go#L258-L262